tutorial/inputoutput.po

```diff
@@ -41,13 +44,18 @@ msgid ""
 "method of file objects; the standard output file can be referenced as ``sys."
 "stdout``. See the Library Reference for more information on this.)"
 msgstr ""
+"目前為止我們已經學過兩種寫值的方式：*運算陳述式* 與 :func:`print` 函式。（第"
```

Comment:
The latter star for the italic markup should have a backslash after it to avoid redundant space in the generated output. Please check out [rST notes in translation guideline](https://github.com/python/python-docs-zh-tw#rst) for the syntax.

Also, I think "expression statement" is better translated to "表示式陳述" since "expression" is not the same concept as "computation", although it could be.  And please quote the original English here for clearness.